game-runner/src/main/java/za/co/entelect/challenge/engine/runner/GameEngineRunner.java

```diff
@@ -159,6 +164,17 @@ public void onRoundComplete() {
                 log.error("Failed to write round information", e);
             }
         }
+
+
+        try {
+            String roundDirectory = FileUtils.getRoundDirectory(gameMap.getCurrentRound());
+            String globalStateFile = String.format("%s/%s/GlobalState.txt", gameRunnerConfig.gameName, roundDirectory);
```

Comment:
we should keep the same case as `state.json` `state.txt` `command.txt`
those are either java case(best case), or the-kebab-case(the css case)

Reply:
The match logs are named: `JsonMap.json`, `PlayerCommand.txt` and `TextMap.txt`. I named it `GlobalState.json` for consistency.

Reply:
oh...why did we? 🤯
too late now. `GlobalState.json` is good then 😂